frontend/src/views/Seedlot/SeedlotRegistrationForm/styles.scss

```diff
@@ -41,3 +42,33 @@
     margin-right: 1rem;
   }
 }
+
+.seedlot-registration-modal {
+
+  .bcgov--modal-content {
```

Comment:
Use `#{vars.$bcgov-prefix}` instead of `bcgov` directly, please fix on other places in this file.